microshift_storage/creating-volume-snapshots-microshift.adoc

```diff
@@ -0,0 +1,89 @@
+:_content-type: ASSEMBLY
+[id="creating-volume-snapshots-microshift"]
+= Creating volume snapshots
+include::_attributes/attributes-microshift.adoc[]
+:context: creating-volume-snapshots-microshift
+
+toc::[]
+
+This document describes how to use volume snapshots with the supported {product-title} Container Storage Interface (CSI) driver to help protect against data loss. Familiarity with xref:../microshift_storage/understanding-persistent-storage-microshift.adoc#persistent-volumes_understanding-persistent-storage-microshift[persistent volumes] is required.
+
+A snapshot represents the state of the storage volume in a cluster at a particular point in time. Volume snapshots can also be used to provision a new volume.
+
+{product-title} supports CSI volume snapshots with the {product-title} CSI driver.
```

Comment:
Since MicroShift is integrated with LVMS CSI driver, may be rephrase "... snapshots with the LVMS CSI driver"